ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-master.yaml

```diff
@@ -0,0 +1,55 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: submariner-addon
+promotion:
+  name: "2.2"
+  namespace: open-cluster-management
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.4"
```

Comment:
Are you sure about using `4.4` for your testing?

Reply:
Thanks, I have updated to 4.6